test/completion/bash-it.completion.bats

```diff
@@ -58,9 +58,9 @@ function __check_completion () {
   assert_line -n 0 "vagrant vault vim"
 }
 
-@test "completion bash-it: update - show no options" {
+@test "completion bash-it: update - show options" {
```

Comment:
Can you please add another unit test for the `-s/--silent` options?

Reply:
done